scripts/tests/large-setup-test.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+
+set -euo pipefail
+export RUST_LOG="${RUST_LOG:-info}"
+
+source scripts/_common.sh
+build_workspace
+add_target_dir_to_path
+make_fm_test_marker
+
+export FM_FED_SIZE=$(((RANDOM % 10) + 4))
```

Comment:
[nit]

The spirit of this test is to ensure DKG and federation setup works with `n` peers. If we want to additionally verify the dev fed setup succeeds in a degraded state, we could add:
```
export FM_OFFLINE_NODES=$(((FM_FED_SIZE - 1) / 3))
```

However, the current default `FM_OFFLINE_NODES=1` doesn't scale well for larger federations (e.g., with 13 peers we could tolerate 4 offline, not just 1). 

I think we should either set `FM_OFFLINE_NODES=0` for this test or set it to the max BFT-tolerant value using the formula above.

https://github.com/fedimint/fedimint/blob/2d5ad92f244622989ecef2f074be338d8267740e/scripts/_common.sh#L100-L101